core-bundle/src/Doctrine/Backup/Backup.php

```diff
@@ -94,6 +95,6 @@ private static function validateFilePath(string $filepath): string
             throw new BackupManagerException(sprintf('The filepath "%s" does not match "%s"', $filepath, self::VALID_BACKUP_NAME_REGEX));
         }
 
-        return $filepath;
+        return Path::normalize($filepath);
```

Comment:
The backup manager receives a base path with forward slashes, but Symfony Finder will then return the file path with backwards slashes under Windows (for the path part below the base path). We can either fix this by normalising the file paths before comparing them in the test - or we normalize them here. I think I would go for the latter.